src/test/java/org/broadinstitute/hellbender/tools/recalibration/CycleCovariateUnitTest.java

```diff
@@ -82,4 +82,33 @@ public void testMaxCyclePasses() {
         ReadCovariates readCovariates = new ReadCovariates(read.getReadLength(), 1);
         covariate.recordValues(read, readCovariates);
     }
+
+    private static int expectedCycleKey(final int baseNumber, final boolean isNegStrand, final boolean isSecondInPair, final int readLength, final boolean indel, final int maxCycle) {
```

Comment:
hmm,  maybe i should add some comment about where this computation is coming from. It's coming from the CycleCovariate. We could refactor CycleCovariate to use the same code as this method.